stacks/api/api.yml

```diff
@@ -0,0 +1,109 @@
+x-env-config: &env-config
+  environment:
+    # -- Runtime
+    - NODE_ENV=${NODE_ENV:-production}
+    - FREECODECAMP_NODE_ENV=${FREECODECAMP_NODE_ENV:-production}
+    - PORT=${PORT:-3000}
+    - HOST=${HOST:-0.0.0.0}
+    # -- Database
+    - MONGOHQ_URL=${MONGOHQ_URL}
+    # -- Logging
+    - FCC_API_LOG_LEVEL=${FCC_API_LOG_LEVEL:-info}
+    - SENTRY_DSN=${SENTRY_DSN}
+    - SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}
+    # -- Debugging
+    - FCC_ENABLE_SENTRY_ROUTES=${FCC_ENABLE_SENTRY_ROUTES:-false}
+    - FCC_ENABLE_SWAGGER_UI=${FCC_ENABLE_SWAGGER_UI:-false}
```

Comment:
If these exist, that is fine, but it nullifies the use of Growthbook, because the env takes precedence over the GB values.

The main reason we wanted GB on the api is so that, when we wanted to change "features", we did not need to manually go into each VM to change the env. Instead, with GB, we can just go to the dashboard, change the values, then restart the api processes - provided the env values are not set.

Reply:
These keys will not be set in production and can be removed from the Docker stack file in future. 

They are added in the PR as a backup - because I am unsure of all the code that is being written/shipped recently.